packages/controllers/header.coffee

```diff
@@ -1,5 +1,13 @@
-Template.header.onCreated ->
-  @accountsState = new ReactiveVar("signIn")
+if Meteor.isClient
+  Template.header.onCreated ->
+    @subscribe('currentUserName', Meteor.userId())
```

Comment:
It would make more sense to put the currentUserName stuff in accountsHeaderButtons if that's the only part that uses it.